x-pack/plugins/cases/server/authorization/index.ts

```diff
@@ -190,6 +190,14 @@ const CaseOperations = {
     docType: 'case',
     savedObjectType: CASE_SAVED_OBJECT,
   },
+  [WriteOperations.AssignCase]: {
+    ecsType: EVENT_TYPES.change,
+    name: WriteOperations.AssignCase as const,
+    action: 'case_assign',
```

Comment:
nit: I know we missed this with `case_reopen`, but I think the pattern they use is the pluralized `cases`